src/component/Contact.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
```

Comment:
you can club Form and Button import as both of them from same lib.
`import { Form, Button } from "react-bootstrap"`